docs/en/setup/service-agent/java-agent/Plugin-list.md

```diff
@@ -25,6 +25,7 @@
 - feign-pathvar-9.x
 - finagle
 - graphql
```

Comment:
```suggestion
```

I think you forgot to remove this.

Reply:
There is still the older versions, graphql 8 and 9

Reply:
Guessing you would like those keys updated too?